NOTICE

```diff
@@ -0,0 +1,6 @@
+The Initial Developer of the code in config/src/ripgrep_config.rs,
+is Andrew Gallant (https://github.com/BurntSushi).
+Copyright (c) 2015 Andrew Gallant
+The code was originally licensed under the MIT license, a copy of
+which can be obtained at https://opensource.org/licenses/MIT
+The code was adapted by Stackable GmbH.
```

Comment:
It might by picky, but you should not call it "Stackable GmbH" as long as you're not yet in Handelsregister

Reply:
Not picky at all, I've changed the company name to "Stackable GmbH i.G."